### PR TITLE
Fix tox -e lint

### PR DESCRIPTION
**Pull request by ale-rt**

### CODE REVIEW

**Comment by ale-rt** [comment]
This fixes linting for #15.
I am not sure I have been doing it right.

File: MANIFEST.in [hunk 1]
```diff
@@ -3,7 +3,9 @@
 include *.md
 include *.rst
 include *.txt
+include .isort.cfg
```
Comment (dataflake) [inline MANIFEST.in:6]: Please move all isort configuration into `setup.cfg` by way of `.meta.toml`, that way we use one consistent location.

File: setup.py [hunk 2]
```diff
@@ -12,15 +12,13 @@
...
-
-import os
-
 from setuptools import find_packages
 from setuptools import setup
 
 
 def read(path):
     """Read the contents of a file system path."""
+    import os
```
Comment (dataflake) [inline setup.py:21]: This is inconsistent with all other projects. Please revert.

File: setup.py [hunk 3]
```diff
@@ -48,8 +46,11 @@ def read(path):
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
+        'Programming Language :: Python :: 3.11',
```
Comment (dataflake) [inline setup.py:53]: The file is missing a `python_requires` as well

File: src/zc/relation/catalog.py [hunk 4]
```diff
@@ -11,19 +11,19 @@
...
-import copy
-import sys
-
-import zope.interface
-import zope.interface.interfaces
-
 import BTrees
 import BTrees.check
 import BTrees.Length
+import copy
```
Comment (dataflake) [inline src/zc/relation/catalog.py:17]: This is inconsistent with all other projects. Please use the `isort` configuration generated by meta/config and move from there.

File: src/zc/relation/searchindex.py [hunk 5]
```diff
@@ -11,12 +11,12 @@
 # FOR A PARTICULAR PURPOSE.
 #
 ##############################################################################
+import BTrees
 import copy
+import persistent
```
Comment (dataflake) [inline src/zc/relation/searchindex.py:16]: See isort comments above

**Review by dataflake — CHANGES_REQUESTED** [review]
The isort configuration used here is inconsistent with what we have been using in all other projects. Please run meta/config on the code to generate the isort config in `setup.cfg` and go from there.

**Comment by ale-rt** [comment]
Thanks @dataflake

**Comment by ale-rt** [comment]
Replaced by #17

**Pull request closed without merging**